### ISSUE TITLE
ansible-xml needs new care takers

### ISSUE THREAD

**Issue by tbielawa** [cmprescott/ansible-xml#16]
Ping to relevant contributors:

* @lx
* @cmprescott
* @relaxnow 
* @dekimsey

Thanks to everyone for showing their interest in this module! The response we've gotten since we first published the (very rough and very incomplete) first version has been more than I ever anticipated.

Sadly, the original maintainers no longer have the time required to give this project the attention it now requires. The time has come for this module to find new care takers who can give it the care it requires. Therefore @RHInception has decided that we are officially end-of-life'ing this project.

From reviewing the most recent comments on the issue tracker, it seems that @Lx and @cmprescott have expressed the most interest in taking over this project. I do not want to personally dictate the future of this module, rather I'd like to create a proper venue (this issue) for arranging a clean hand-over of the project.

# Open Pull Requests

All PR's which were open this morning have been merged, de-conflicted, and pushed.

# Open Issues

After merging all open PR's I manually closed issue #13 (tilde-expansion in file paths)

The remaining open issues will not be addressed by myself or the rest of the RH Inception team and are left as an exercise to the new future maintainers

# Next Steps

I'd love to see a contributor step up and state their intention to officially take over ownership of the project. We want to leave the future of this module in the hands of those who still have the energy required to do the needful.

Once consensus has been reached I'll be happy to arrange to transfer ownership of the project to the new maintainer(s). We will not interfere if a consensus cannot be reached, rather the project will go into read-only mode and refer new users to the [forks](https://github.com/RHInception/ansible-xml/network) chart.

# ?

Any takers? (thoughts?)

**Comment by Lx** [comment]
@cmprescott, I understand from [your comment in issue #12](https://github.com/RHInception/ansible-xml/issues/12#issuecomment-98441506) that you're definitely interested in taking ownership of this project.

No contest from me; I can't commit at this time to any regular maintenance.

**Comment by cmprescott** [comment]
I'd love to take it over with the ultimate goal of getting it merged into [ansible-modules-extras](https://github.com/ansible/ansible-modules-extras). I'm in a position where I'd have a few hours a week to devote to the project. I'd be happy to share the responsibilities with anyone else who would like it though.

Thanks,
Chris

**Comment by tbielawa** [comment]
Well that was easy

@cmprescott I'm going to transfer ownership over to you now.

**Comment by tbielawa** [comment]
GitHub told me I can only transfer from an org to myself at this time. I am doing that now.

    Moving repository to tbielawa/ansible-xml. This may take a few minutes...

Once that is complete I'll try transferring to @cmprescott again. You will receive an email from GitHub allowing you to accept to decline the transfer.

Additional information about transfers is available on this page: https://help.github.com/articles/transferring-a-repository/

**Comment by tbielawa** [comment]
@cmprescott done

    Repository transfer to cmprescott requested

**Comment by relaxnow** [comment]
I see that the move was successful. Nice.
FYI: I donated some time to this module because I needed this for a project, but don't work on this actively anymore and don't foresee using it in the near future.
Wishing @cmprescott happy hacking :smile:
